chain/client/src/client_actions.rs

```diff
@@ -95,8 +95,8 @@ pub struct SyncJobsSenderForClient {
     pub resharding: Sender<ReshardingRequest>,
 }
 
-pub struct StateWitnessDistributionSenderForClient {
-    pub distribute_chunk_state_witness: Sender<DistributeChunkStateWitnessRequest>,
+pub struct StateWitnessSenderForClient {
```

Comment:
Are there other StateWitnessSenders? If not I would rename further to just `StateWitnessSender` and move it to the same file where the request is defined.

Reply:
There's one for Client and one for Network (and possibly more in future)